community/README.md

```diff
@@ -0,0 +1,42 @@
+# Welcome to OpenEBS Community
+
+This is a meta-repository for OpenEBS. Here, you will find links to various artifacts related to documentation, e2e tests and code related to building and deploying OpenEBS.
+
+Nevertheless, below find a list of many general channels, groups and meetings devoted to OpenEBS.
+
+## Communication
+
+- [Slack](https://slack.openebs.io) - Sign Up.
+
+- [Blog](https://blog.openebs.io) - Blogs on Containerized Storage and DEVOPS.
+
+- [Twitter](https://twitter.com/openebs) - Follow.
+
+Real time discussions happen at [OpenEBS Community Slack](https://openebs-community.slack.com/).
+
+## How Can I Help?
+
+Documentation (like the text you are reading now) can always use improvement!
+
+There's a curated list of issues, [Good First Issue](https://github.com/openebs/openebs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) that should not need deep knowledge of the system.
+
+To dig deeper, please start by having a look at [OpenEBS Architecture](../contribute/design/README.md).
+
+There's always code that can be clarified and variables or functions that can be renamed or commented.
+
+There's always a need for more test coverage that by contributed through [OpenEBS E2E](../contribute/CONTRIBUTING-TO-CI-AND-E2E.md
+).
+
+## Issues
+
+If you have a question about OpenEBS or have a problem using it,
+please start with the [troubleshooting guide](http://openebs.readthedocs.io/en/latest/troubleshooting/troubleshooting.html).
+
+If that doesn't answer your questions, or if you think you found a bug,
+please [file an issue](https://github.com/openebs/openebs/issues).
+
+## Weekly Meeting
+
+We have PUBLIC [weekly meeting](https://zoom.us/j/438333946) every Friday at 10am Indian Standard Time over Zoom.
+
+Map that to your local time with this [timezone table](https://www.google.com/search?q=1000+am+in+ist).
```

Comment:
How about using http://www.thetimezoneconverter.com/ here rather than google link. So that everyone can covert to their timezone.

Reply:
@dungeonmaster18, google shows the meeting time in their timezone. They do not need to convert it to their timezone.